dmalloc_t.c

```diff
@@ -1860,8 +1863,9 @@ static	int	check_special(void)
   void	*pnt;
   int	page_size;
   int	final = 1;
-  char	*old_env, env_buf[256];
-  
+  char	*old_env;
+  static char	env_buf[256];
```

Comment:
256 bytes is  acceptable even in the small RTOS. the default stack size for RTOS is normally at least 1KB or 2KB.